addons/MetroidvaniaSystem/Scripts/Settings.gd

```diff
@@ -6,7 +6,7 @@ extends Resource
 	set(t):
 		if t == theme:
 			return
-		
```

Comment:
Can you remove the trimming?
I'm not using it, so it just creates unnecessary noise. I'd probably restore the tabs anyway.